x/staking/client/cli/flags.go

```diff
@@ -30,6 +30,7 @@ const (
 	FlagGenesisFormat = "genesis-format"
 	FlagNodeID        = "node-id"
 	FlagIP            = "ip"
+	FlagHomeServer    = "home-server"
```

Comment:
```suggestion
	FlagHomeServer    = "server-home"
```

sounds a tad better IMvHO

Reply:
I was aiming for symmetry with the old `--home-client=`, but yes, you're right, `--server-home` is better.